server/src/main/scala/za/co/absa/atum/server/model/PartitioningForDB.scala

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.model
+
+import za.co.absa.atum.model.dto.PartitionDTO
+
+private[server] case class PartitioningForDB private(
```

Comment:
I meant making the constructor private, didn't need to make the whole class so. But almost as good. 😄

Reply:
It is done